src/Robo/Plugin/Traits/DatabaseDownloadTrait.php

```diff
@@ -141,7 +141,7 @@ protected function s3BucketRequestConfig(string $siteName): array
      */
     protected function s3BucketForSite(string $siteName): string
     {
-        if (!$bucket = $this->getConfig('database_s3_bucket', $siteName)) {
+        if (!is_string($bucket = $this->getConfig('database_s3_bucket', $siteName))) {
```

Comment:
Same concern there. The previous line is better, if you can get getConfig() to _behave_.